### PR TITLE
docs: Add page for encrypted messages

### PR DESCRIPTION
**Pull request by krille-chan**
This is more or less just copy pasted from fluffychat as a placeholder until we have a real helpcenter page for this topic. I just need something to link

Closes https://github.com/famedly/app/issues/5296

### CODE REVIEW

File: docs/help-center/using-famedly/encrypted-messages.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: Encrypted Messages
+title: Encrypted Messages
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 1
```
[the remark this answers is not in the document]
Comment (krille-chan) [reply docs/help-center/using-famedly/encrypted-messages.mdx]: I set it to 7 for now 🤔 is this ok?

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/encrypted-messages.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Verschlüsselte Nachrichten
+title: Verschlüsselte Nachrichten
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 1
```
[the remark this answers is not in the document]
Comment (krille-chan) [reply i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/encrypted-messages.mdx]: Done :)

File: docs/help-center/using-famedly/encrypted-messages.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+It is also possible that the sender has blocked your device or something
+went wrong with the internet connection.
+
+Are you able to read the message on another
+session? Then you can transfer the message from it! Go to Settings > Devices and make sure
+that your devices have verified each other. When you open the room the next time and both
+sessions are in the foreground, the keys will be transmitted automatically.
+
+Do you not want to lose the keys when logging out or switching devices? Make sure that you have enabled
+the chat backup and do not loose your passphrase.
```
[the remark this answers is not in the document]
Comment (krille-chan) [reply docs/help-center/using-famedly/encrypted-messages.mdx]: Done

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/encrypted-messages.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Login</h2>
+    <p class="hero__subtitle">Wie umgehen mit verschlüsselten Nachrichten in Famedly</p>
+  </div>
+</div>
+
+## Why are my messages encrypted and unreadable?
```
[the remark this answers is not in the document]
Comment (krille-chan) [reply i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/encrypted-messages.mdx]: Done

**Review by mErrenst — APPROVED** [review]
Thanks for reworking :D

**Pull request merged**